solidity/contracts/KeepRandomBeaconOperator.sol

```diff
@@ -143,17 +142,14 @@ contract KeepRandomBeaconOperator is ReentrancyGuard {
         startGroupSelection(_genesisGroupSeed, msg.value);
     }
 
-    /**
-     * @dev Throws if called by any account other than the owner.
-     */
-    modifier onlyOwner() {
-        require(owner == msg.sender, "Caller is not the owner");
+    modifier onlyServiceContractKeeper() {
```

Comment:
Are we sure we want to introduce the term "keeper" anywhere? It already has a meaning [in the cryptocurrency space](https://medium.com/@rzurrer/keepers-workers-that-maintain-blockchain-networks-a40182615b66) and it's easy to confuse with "keep"

Reply:
is `serviceContractUpgrader` a good name?  similar to existing `operatorContractUpgrader` that can add operator contracts to service contract so `serviceContractUpgrader` in this case adds service contracts to operator contract

Reply:
We already have `registryKeeper` in the `Registry` but I do agree and my fingers paused for a minute before typing it. I have no better idea how to name it but I am of course open for suggestions, renaming it is two minutes of work :)

Reply:
Good idea @ngrinkevich, let's go with `serviceContractUpgrader`. This guy is doing the same as `operatorContractUpgrader` for the service contract.